coinstacks/ethereum/ingester/src/workers/txid.ts

```diff
@@ -10,16 +10,18 @@ if (!INDEXER_WS_URL) throw new Error('INDEXER_WS_URL env var not set')
 
 const blockbook = new Blockbook({ httpURL: INDEXER_URL, wsURL: INDEXER_WS_URL })
 
+const msgLogger = logger.child({ namespace: ['workers', 'txid'], fn: 'onMessage' })
 const onMessage = (worker: Worker) => async (message: Message) => {
   const txid: string = message.getContent()
 
   try {
     const tx = await blockbook.getTransaction(txid)
+    msgLogger.debug({ txid, tx }, 'getTransaction')
 
     worker.sendMessage(new Message(tx), 'tx')
     worker.ackMessage(message, txid)
   } catch (err) {
-    logger.error('onMessage.error:', err.isAxiosError ? err.message : err)
+    msgLogger.error(err, 'Error processing transaction')
```

Comment:
I had the extra check on the axios errors because logging the actual full axios error is majorly verbose and not useful

Reply:
the logger has a built-in formatter for errors.

Reply:
Sweet, that is good to know